beetsplug/fish.py

```diff
@@ -127,18 +127,10 @@ def run(self, lib, opts, args):
         totstring += get_cmds_list([name[0] for name in cmd_names_help])
         totstring += "" if nobasicfields else get_standard_fields(fields)
         totstring += get_extravalues(lib, extravalues) if extravalues else ""
-        totstring += (
-            "\n"
-            + "# ====== {} =====".format("setup basic beet completion")
-            + "\n" * 2
-        )
+        totstring += "\n" "# ====== setup basic beet completion =====" "\n\n"
         totstring += get_basic_beet_options()
         totstring += (
-            "\n"
-            + "# ====== {} =====".format(
-                "setup field completion for subcommands"
-            )
-            + "\n"
+            "\n" "# ====== setup field completion for subcommands =====" "\n"
```

Comment:
I don't get why these have to be separate strings in the new version, this could be simplified.

Reply:
Oh, yeah, good catch!  I think I just moved past them too quickly to pay attention.

Reply:
Actually, I think I'm just going to submit a whole rewrite for this module.  There's many more instances of bad string building in here that I'd like to address more comprehensively with its own PR.  Can we ignore this for now?

Reply:
Perhaps for this PR, just put them in as one string, and then in another PR change it however you think is acceptable and I'll review then. This change just looks strange at the moment and I worry someone will think there's a reason for it to be like this.